conf/application.conf

```diff
@@ -139,6 +144,12 @@ gg {
   origin = "cgtpd"
 }
 
+external-url {
+  gov-call-charges =  "https://www.gov.uk/call-charges"
+  additional-needs = "https://www.gov.uk/dealing-hmrc-additional-needs/overview"
+  trust-registration = "https://www.tax.service.gov.uk/trusts-forms/form/registration-of-a-trust/new"
+}
+
```

Comment:
how do these urls relate to IV?

Reply:
the call charges url and the additional needs url's are used for the insufficient evidence page. The trust registration url isn't used for iv - it's used to direct users to the trust registration service when they need to go and register a trust